velox/functions/lib/tests/DateTimeFormatterTest.cpp

```diff
@@ -2192,4 +2200,21 @@ TEST_F(MysqlDateTimeTest, parseFractionOfSecond) {
   EXPECT_THROW(parse("9999999", "%f", false), VeloxUserError);
 }
 
+class FmtDateFormatterTest : public DateTimeFormatterTest {};
+
+TEST_F(FmtDateFormatterTest, fmtDateFormatter) {
```

Comment:
Let's not mixup different formatting concepts and move this test somewhere else. Is there already a test file for Date type? If not, let's add DateTest.cpp with DateTest test class and add TEST_F(DateTest, format).